scripts/upgrade

```diff
@@ -30,6 +30,10 @@ if [ -z "${userpass:-}" ]; then
     ynh_app_setting_delete --app="$app" --key="password"
 fi
 
+if [ -z "${domain:-}" ]; then
+    ynh_die --message="This new version requires a domain, please remove and reinstall."
```

Comment:
maybe tell the admin to set a domain and path by hand in a `doc/PRE_UPGRADE.d/2.0.14~ynh3.md`?

```bash
yunohost app setting mosquitto domain -v mqtt.example.com
yunohost app setting mosquitto -v /example
```

and put here an error message to ask for it again?